docs/contributing.md

```diff
@@ -20,96 +20,117 @@ git clone https://github.com/executablebooks/sphinx-book-theme
 cd sphinx-book-theme
 ```
 
-To work on this project, you need Python 3.6 or newer. Most of this project's
-development workflow commands use `nox`, which can be installed with pip:
+To work on this project, you need Python 3.6 or newer.
+Although you can create your own development environment,
+the recommended way to work on development is *via* [tox](https://tox.readthedocs.io), which can be installed with pip:
 
-```bash
-pip install nox
+```console
+$ pip install tox
+# list all environments
+$ tox -a
 ```
 
-### Building documentation
+If you have Conda installed you may also consider installing the plugin:
 
-You can now build documentation with your local copy of this theme! Try it
-out on the theme's documentation like so:
-
-```bash
-nox -s docs
+```console
+$ pip install tox-conda
 ```
 
-This will generate the HTML documentation and compile the relevant stylesheets.
-The generated documentation which can be viewed by opening
-`docs/_build/html/index.html` in your browser.
+See the `sphinx-book-theme/tox.ini` for details of all available development environments.
 
 ### Running Tests
 
 This theme has a test suite to ensure that all the relevant user content is
 correctly handled. The tests can be run using:
 
-```bash
-nox -s tests
+```console
+$ tox
 ```
 
-This will run tests against all supported version of Python that are installed.
+You can also run against different Python and sphinx versions, and supply arguments to pytest:
 
-If you want to run tests for a specific version of Python (say, 3.8), you can
-do so using:
+```console
+$ tox -e py38-sphinx2 -- -x
+```
 
-```bash
-nox -s tests-3.8
+:::{tip}
+To "re-build" an environment, wih updated dependancy versions, use:
+
+```console
+$ tox -r -e py38-sphinx2
 ```
 
-### Running Linters
+:::
 
-The code style in this project is enforced with multiple automated linters. You
-can run them using:
+### Running Pre-commit
 
-```bash
-nox -s lint
+The consistency and code style in this project is enforced with multiple automated [pre-commit](https://pre-commit.com) hooks.
+You can run them using:
+
+```console
+$ tox -e py38-pre-commit -- --all
 ```
 
-### Working on the theme "live"
+or directly (after installing pre-commit):
 
-If you're making changes to your local copy of this theme, there is a helper
-command :
+```console
+$ pre-commit run --all
+$ pre-commit install
+```
 
-```bash
-nox -s docs-live
+### Working on the theme
```

Comment:
can we document `theme_dev_mode`

Reply:
yeh, as mentioned in the code comments, this option should literally only be used in this repos conf.py; to allow for the CSS/JS to be dynamically updated